src/App.js

```diff
@@ -1,10 +1,12 @@
-import React from "react";
+import React from 'react';
+import { Provider } from 'react-redux';
+import store from './redux/store';
 
-const AppComp = () => {
-  return (
+const AppComp = () => (
+  <Provider store={store}>
     <div>
       <h2>this is indiana front-end pages kelvin</h2>
```

Comment:
Okay so I was ignoring this message before but then really? `this is indiana front-end pages kelvin`? I know you didn't add it @primuse but kindly update it biko.

Reply:
Okay @p-funky